src/grt/grt-avhpi.adb

```diff
@@ -27,6 +27,19 @@ with Grt.Vstrings; use Grt.Vstrings;
 with Grt.Rtis_Utils; use Grt.Rtis_Utils;
 with Grt.To_Strings;
 
+-- Acronyms that I don't understand:
+   -- Rti
+   -- Rtis
+   -- Rtin
+
+
+-- Questoins:
+   -- Explanation for the Ctxt objects would be really helpful.
+   -- Why are they used, and what do they represent?
+   -- Does ghdl support a VHPI interface?
```

Comment:
Not at the C level.  There is AVHPI which is inspired from VHPI but uses Ada style.  But there is no C VHPI interface.  Shouldn't be very hard to do, but there are very few examples.